torch/csrc/jit/script/module.h

```diff
@@ -470,6 +470,20 @@ struct Module {
   bool optimize;
 };
 
+static inline void recursivelyCallMethod(std::shared_ptr<Module> mod, const std::string& method_name) {
```

Comment:
Can we just make this the equivalent of the Python modules apply function? C++ code would then be similar for unpack:

```
void script::Module::apply(std::function<void(Module&)> fn);
```

```
m.apply([](Module& m) {
  m.get_method("_unpack")({});
});
```